### ISSUE TITLE
Error when creating new file

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#933]
I used an internal IBM build from today (shows Sept  6 as the date in About Maqetta), created an Eclipse-compatible project, and then issued a New HTML file command in Win/FF. The command stops midway with the following exception showing in Firebug console:

```
davinci.ve.metadata.getLibrary(r.$library) is undefined
[Break On This Error] var _929=davinci.ve.metadata.getLibrary(r.$library).version; 
```

**Comment by JonFerraiolo** [comment]
Updated title - I just tried the latest build on maqetta.org/nightly (http://maqetta.org/nightly/maqetta-20110906-0919.zip) and I can't do a New File even for default project, whether Chrome or FF

**Comment by JonFerraiolo** [comment]
Works OK in dev mode, just not in builds.

**Comment by childsb** [comment]
Fixed with checking c6ec77d138a1c9901170f9e366961d70b07a1779
